docs/content/configuration.md

```diff
@@ -925,3 +929,13 @@ with data put into the configured `directory`.
 | `storage.disk.badger` | `string` | No (default: empty) | "Superflags" passed to Badger allowing to modify advanced options. |
 
 See [the docs on disk storage](../misc-disk/) for details about the settings.
+
+### Server
+
+The `server` configuration sets the gzip compression settings for `/v0/data`, `/v1/data` and `/v1/compile` HTTP `POST` endpoints
+The gzip compression settings are used when the client sends `Accept-Encoding: gzip`
+
+| Field                           | Type  | Required            | Description                                                                                                                                                                                                                                  |
+|---------------------------------|-------|---------------------|----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| `server.gzip_min_length`        | `int` | No, (default: 1024) | Specifies the minimum length of the response to compress                                                                                                                                                                                     |
+| `server.gzip_compression_level` | `int` | No, (default: 9)    | Specifies the compression level. Default set to `9` (`gzip.BestCompression`). Accepted values: `gzip.NoCompression`, `gzip.BestSpeed`, `gzip.BestCompression`, `gzip.DefaultCompression`. See https://pkg.go.dev/compress/flate#pkg-constants |
```

Comment:
These constants can't be used directly in the configuration, right? If so, I'd just say something like "a value of either 0 (no compression), 1 (best speed, lowest compression) or 9 (slowest, best compression)".

We can keep the link though, as it's rather informative.

Reply:
Yes, the constants can't be used directly. Good wording, fixed.